src/Tizen.NUI/src/public/BaseComponents/TextField.cs

```diff
@@ -1475,6 +1548,48 @@ private void SystemSettings_LocaleLanguageChanged(object sender, LocaleLanguageC
             }
         }
 
+        private void SystemSettings_FontSizeChanged(object sender, FontSizeChangedEventArgs e)
```

Comment:
Please avoid to use `_` for the method name.
NUI is not using `_` for the new code since last version.

Reply:
I'll remove "_" from the method name.